Assets/MixedRealityToolkit.Examples/Demos/ExamplesHub/Scripts/ExamplesHubHomeButtonActivation.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.MixedReality.Toolkit;
```

Comment:
nit: add license header. This has recently changed to the following:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

Reply:
Updated license header.